### PR TITLE
Bug 1512484 - Allow to specify default version per product

### PR DESCRIPTION
**Pull request by kyoshino**
Add `default_version` to products, and make it editable by admin. The implementation is almost the same as Default Milestone. Also make the field optional when filing a bug (especially via the API.)

## Bugzilla link

[Bug 1512484 - Allow to specify default version per product](https://bugzilla.mozilla.org/show_bug.cgi?id=1512484)

### CODE REVIEW

**Comment by kyoshino** [comment]
Note to myself: this requires a separate schema-only PR.

File: Bugzilla/WebService/Constants.pm [hunk 1]
```diff
@@ -209,8 +209,10 @@ use constant WS_ERROR_CODE => {
   product_name_already_in_use          => 702,
   product_name_diff_in_case            => 702,
   product_must_have_description        => 703,
-  product_must_have_version            => 704,
+  # Removed in favour of default value 'unspecified':
```
Comment (dklawren) [inline Bugzilla/WebService/Constants.pm]: Just remove the two lines

**Comment by kyoshino** [comment]
Thanks @dklawren 😺 Will solve the conflict and you feedback, and send a schema-only PR. Once this is shipped, the default version for the BMO product can be Production instead of Stage, which actually annoys us a bit.

**Comment by kyoshino** [comment]
Solved the conflict in `DB.pm`.

**Pull request closed without merging**